docs/.pages

```diff
@@ -6,4 +6,5 @@ nav:
   - howto
 #  - tutorials
   - reference
+  - explanation
```

Comment:
Creating a new section isn't necessary here, the `concepts` section is already there for explainers. Though, rereading <https://diataxis.fr/explanation/>, maybe "Background" would be an even better name?

Reply:
I too, refreshed my memory regarding the Diataxis categories and realized that this post looks like "Explanation" material. Still, something bugs me regarding this particular categorization, so yes, let's go with "Background". Or we could also go with "Discussion". What do you think?

Reply:
I think "Background" is good. So, let's replace `concepts` with `background`, and we'll let this addition be the inaugural "Background" piece.

Reply:
The only question is where "Background" goes in the pecking order (i.e. in `docs/.pages`): I think it should definitely go after "Tutorials", but does it come before or after "Reference"?

Reply:
Since I view "References" as the last place any self-respecting user who values their time and loves life in general checks, I believe "Backgroud" should go **before** that.

Reply:
Sounds good to me! Please update your branch accordingly. :)